TEAMS.md

```diff
@@ -0,0 +1,59 @@
+# Projects details
+
+| *Project*       | *Maintainers*  | *Teams* |
+| --------------- | -------------- | ------- |
+[ZendSkeletonApplication](https://github.com/zendframework/ZendSkeletonApplication) | [Akrabat](https://github.com/akrabat), [EvantDotPro](https://github.com/EvanDotPro) | community-review-team |
```

Comment:
Same for contributors - use `@username`; it's shorter and creates the links for you.